### PR TITLE
add responsibleai curated environment to azureml-assets

### PR DESCRIPTION
**Pull request by imatiach-msft**
Add curated environment for raiwidgets and responsibleai 0.22 release.

Link to release notes:
https://github.com/microsoft/responsible-ai-toolbox/releases/tag/v0.22.0

### CODE REVIEW

File: training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: sklearn-1.1
```
Comment (elesel) [inline training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:1]:
```suggestion
name: responsibleai
```

File: training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+
+description: >-
+  AzureML Responsible AI curated image.
```
Comment (elesel) [inline training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/spec.yaml:4]:
```suggestion
  AzureML Responsible AI environment.
```

**Review by elesel — CHANGES_REQUESTED** [review]
Would you please consider my recommendations? Looks like you're almost there.

File: training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml (hunk 1 above)
Comment (imatiach-msft) [reply training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml:1]: done!

File: training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/spec.yaml (hunk 2 above)
Comment (imatiach-msft) [reply training/general/src/environments/responsibleai-ubuntu20.04-py38-cpu/spec.yaml:4]: done!

**Review by elesel — APPROVED** [review]

**Review by vizhur — APPROVED** [review]

**Pull request merged**